services/node_create.proto

```diff
@@ -27,62 +25,100 @@ option java_multiple_files = true;
+ * - The newly created node information SHALL be added to the network address
+ *   book information in the network state.
+ * - The new entry SHALL be created in "state" but SHALL NOT participate in
+ *   network consensus and SHALL NOT be present in network "configuration"
+ *   until the next "upgrade" transaction (as noted below).
+ * - All new address book entries SHALL be added to the active network
+ *   configuration during the next `freeze` transaction with the field
+ *   `freeze_type` set to `PREPARE_UPGRADE`.
+ *
+ * ### Record Stream Effects
+ * Upon completion the newly assigned `node_id` SHALL be in the transaction
+ * receipt.
  */
 message NodeCreateTransactionBody {
-
     /**
-     * Node account id, mandatory field, ALIAS is not allowed, only ACCOUNT_NUM.
-     * If account_id does not exist, it will reject the transaction.
-     * Multiple nodes can have the same account_id.
+     * A Node account identifier.
+     * <p>
+     * This account identifier MUST be in the "account number" form.<br/>
+     * This account identifier MUST NOT use the alias field.<br/>
+     * If the identified account does not exist, this transaction SHALL fail.<br/>
+     * Multiple nodes MAY share the same node account.<br/>
+     * This field is REQUIRED.
      */
-    AccountID account_id = 1;
+    proto.AccountID account_id = 1;
 
     /**
-     * Description of the node with UTF-8 encoding up to 100 bytes, optional field.
+     * A short description of the node.
+     * <p>
+     * This value, if set, MUST NOT exceed 100 bytes when encoded as UTF-8.<br/>
+     * This field is OPTIONAL.
      */
     string description = 2;
 
     /**
-     * Ip address and port, mandatory field. Fully qualified domain name is
-     * not allowed here. Maximum number of these endpoints is 10.
-     * The first in the list is used as the Internal IP address in config.txt,
-     * the second in the list is used as the External IP address in config.txt,
-     * the rest of IP addresses are ignored for DAB phase 2.
+     * A list of service endpoints for gossip.
+     * <p>
+     * These endpoints SHALL represent the published endpoints to which other
+     * consensus nodes may _gossip_ transactions.<br/>
+     * These endpoints MUST specify address and port.<br/>
+     * This list MUST NOT be empty.<br/>
+     * This list MUST NOT contain more than `10` entries.<br/>
+     * The first entry in this list SHALL be the endpoint published to all
+     * consensus nodes.<br/>
+     * All other entries SHALL be reserved for future use.
+     * <p>
+     * Some networks restrict these endpoints further.<br/>
+     * A network may require that these endpoints must not specify a DNS name,
+     * may permit a DNS name, or may require a DNS name. The specific
+     * requirements for each network MUST be public, and SHALL apply to all
+     * interactions with the network address book for that network.
```

Comment:
Don't want to loss these details:
The first in the list is used as the Internal IP address in config.txt,
 the second in the list is used as the External IP address in config.txt,
  the rest of IP addresses are ignored for DAB phase 2.

Reply:
Those are specific to our implementation, and would belong in a design document, but not here in the protobuf.
I can note the internal/external part (not config.txt, however) if we believe it will remain a _requirement_ for all implementations.

Reply:
I will copy them to java file comments.